dask_kubernetes/helm.py

```diff
@@ -236,7 +236,7 @@ async def _():
 
         return _().__await__()
 
-    def scale(self, n_workers):
+    def scale(self, n_workers, worker_name=None):
```

Comment:
@jacobtomlinson Would you prefer me to call this `group_name` instead of `worker_name`?

Reply:
Yeah I think `group_name` makes sense here.